lib/brainstem/query_strategies/base_strategy.rb

```diff
@@ -64,7 +73,7 @@ def use_calc_row?
       end
 
       def delegate_count_to_presenter?
-        primary_presenter.evaluate_count?
+        primary_presenter&.evaluate_count?
```

Comment:
Can we avoid returning the nullable boolean? A method that has a question mark should return a true or false. It should not return a nil.

Reply:
On further review, what did we change that made the `primary_presenter` nullable? It doesn't seem like we made any change that would require that.

Reply:
Good catch. This was a quick add to fix an existing failing spec, and I didn't pay much attention to it.